reconcile/utils/terrascript_client.py

```diff
@@ -3027,7 +3027,7 @@ def get_dependencies(tf_resources: Iterable[Resource]
     @staticmethod
     def validate_elasticsearch_version(version):
         """ Validate ElasticSearch version. """
-        return version in [7.7, 7.4, 7.1,
+        return version in [7.10, 7.9, 7.8, 7.7, 7.4, 7.1,
```

Comment:
isnt it better to have it as a global env?

Reply:
its better not to have it here at all :)